packages/website/docs/contributing/intro.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: Introduction
+sidebar_position: 0
+---
+
+# Contributing to Unovis
+
+We want to make contributing to this project as easy and transparent.
+
+## Code of Conduct
+
+F5 has adopted a Code of Conduct that we expect project participants to adhere
+to. Please read [the full text](https://github.com/f5/unovis/blob/main/CODE_OF_CONDUCT.md) so that you can understand
+what actions will and will not be tolerated.
+
+## Our Development Process
+
+Adding a new component:
+1. Create your new component in the TypeScript version of _Unovis_ (`@unovis/ts`);
+2. Generate or manually create wrappers for React, Angular and Svelte;
+3. Write documentation for your new component;
+4. Add examples to Gallery.
+
+For more info, see our [in-depth guide](./guides/adding-a-component).
+
+## Pull Requests
+We actively welcome pull requests. If you want to submit one, please follow the following process:
+
+1. Fork the repo and create your branch from `develop`;
+2. If you've added a new feature, changed the configuration or public methods, please update the docs;
+3. Make sure your code lints;
+4. Submit your PR!
+
+* And finally, if you haven't already, complete the Contributor License Agreement ("CLA") so
+we can merge your code.
+
+## Contributor License Agreement ("CLA")
+
+In order to accept your pull request, we need you to submit a CLA, if you
+haven’t submitted one before for this project. You only need to do this
+once and you’re good to go.
+
+Complete your [CLA here](https://github.com/f5/unovis/blob/main/F5%20Contributor%20License%20Agreement.docx)
+and email the filled out form to _cla@f5.com_.
```

Comment:
Instead of applying _italic_ format, the underscore character become parts of the email address when the page is rendered:
 <img width="215" alt="image" src="https://github.com/f5/unovis/assets/755708/1a53cad5-5afb-403c-aabd-b70c2e4fcbe2">